src/ensembl/src/shared/components/instant-download/instant-download-gene/InstantDownloadGene.tsx

```diff
@@ -135,11 +147,12 @@ const InstantDownloadGene = (props: Props) => {
         theme={props.theme}
         onChange={onTranscriptOptionChange}
       />
-      <InstantDownloadButton
-        className={isButtonDisabled ? styles.downloadButtonDisabled : undefined}
-        isDisabled={isButtonDisabled}
-        onClick={onSubmit}
-      />
+      <div className={styles.downloadButton}>
```

Comment:
Any reason why this class isn't passed directly to `InstantDownloadButton` as a prop?

Reply:
If I pass it to the component, it will only apply to that button. In this case I want to align the whole component to the right side. Is there any other work around?

Reply:
Right. I see the problem. My fault: when I created the `LoadingButton` component, I did not add a way to style the wrapper div around the button [here](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/shared/components/loading-button/ControlledLoadingButton.tsx#L47). We can ignore this for now, I guess, but I wonder if it's going to be a problem for us in the future.

Reply:
create ticket to fix this? @azangru